packages/apple-pay-integration/src/apple-pay-button-initialize-options.ts

```diff
@@ -1,10 +1,23 @@
+import { BuyNowCartRequestBody } from '@bigcommerce/checkout-sdk/payment-integration-api';
```

Comment:
Add an empty string after this line please

Reply:
added